crates/cli/tests/wasm-bindgen/reference.rs

```diff
@@ -59,8 +59,10 @@ use std::env;
 use std::fs;
 use std::path::{Path, PathBuf};
 use std::sync::LazyLock;
+use wasm_bindgen_cli_support::testing::walrus::passes::gc::run as walrus_gc_run;
```

Comment:
Not a fan of renaming imports if avoidable. Lets just import `walrus` so we can use it below.

Reply:
Undone!